Tasks/AzureTestPlanV0/_buildConfigs/Node20/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "azure-tasks-runTestPlan",
+  "version": "1.0.0",
+  "description": "Azure Pipelines Run Automated Test execution Task",
```

Comment:
Should we change the description? We are providing it to execute manual tests as well

Reply:
updated